### ISSUE TITLE
GET /token does not work

### ISSUE THREAD

**Issue by 23Skidoo** [genomizer/genomizer-server#303]
Request:

```
GET /token
Authorization: <skip>
```

Response:

```
     [java] NEW EXCHANGE: GET /token
     [java] Trying to authenticate token  <skip>...
     [java] User testuser authenticated successfully.
     [java] Request body: 
     [java] 
     [java] Response body: 
     [java] {"message":"Could not create command from request"}
```

```
500 Internal Server Error
```

Expected response:

```
200 OK
```

**Comment by 23Skidoo** [comment]
Error 500 happens even when trying to authorize with wrong token (should be 401).

**Comment by parrexion** [comment]
As this command is deprecated there will probably not be a fix to this.
Instead, it should probably be removed the next sprint.

**Comment by 23Skidoo** [comment]
Not sure why it needs to be removed, can be useful for testing. But whatever.

**Comment by 23Skidoo** [comment]
Fixed in #305.
